src/Orchestration/Orchestration.php

```diff
@@ -34,43 +88,57 @@ public function pull(string $image): bool
     /**
      * List Containers
      *
-     * @return bool
+     * @return array
      */
-    public function list(): bool
+    public function list(): array
     {
         return $this->adapter->list();
     }
 
     /**
      * Run Container
+     * 
+     * @param string $image
+     * @param string $name
+     * @param string $entrypoint
+     * @param array $command
+     * @param string $workdir
+     * @param array $volumes
+     * @param array $vars
+     * @param string $mountFolder
+     * @return bool
      */
-    public function run(string $image, string $name, string $entrypoint = '', string $command = '', string $workdir = '/', array $volumes = [], array $vars = []): bool
+    public function run(string $image, string $name, string $entrypoint = '', array $command = [], string $workdir = '/', array $volumes = [], array $vars = [], string $mountFolder = '', array $labels = []): bool
     {
-        return $this->adapter->run($image, $name, $entrypoint, $command, $workdir, $volumes, $vars);
+        return $this->adapter->run($image, $name, $entrypoint, $command, $workdir, $volumes, $vars, $mountFolder, $labels);
     }
 
     /**
      * Execute Container
      *
-     * @param  mixed $name
-     * @param  mixed $command
-     * @param  mixed $vars
+     * @param string $name
+     * @param array $command
+     * @param string $stdout
```

Comment:
Can we add to the docblock that `$stdout` and `$stderr` are passed by reference? 
https://stackoverflow.com/a/36549165